plugins/GSdx/GSdx.rc

```diff
@@ -142,6 +142,7 @@ BEGIN
     CONTROL         "Alpha",IDC_ALPHAHACK,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,14,71,34,8
     CONTROL         "Alpha Stencil",IDC_ALPHASTENCIL,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,108,71,57,8
     CONTROL         "Disable Depth Emulation",IDC_TC_DEPTH,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,108,84,92,8
+    CONTROL         "CPU 8-bit FB Conversion",IDC_8BIT_CPU_FBHACK,"Button",BS_AUTOCHECKBOX | WS_TABSTOP, 14, 84, 92, 8
```

Comment:
There are still spaces after some of the commas.

Reply:
Oh, I completely missed those.